### PR TITLE
Add precompiles weight to DAME

### PR DESCRIPTION
**Pull request by magecnion**

### CODE REVIEW

File: pallets/asset-metadata-extender/src/benchmarking.rs [hunk 1]
```diff
@@ -34,6 +124,203 @@ mod benchmarks {
...
+		u: Linear<0, { <T as Config>::MaxUniversalLocationLength::get() }>,
+	) {
+		let mut handle = MockHandle::new();
+
+		let caller: T::AccountId = whitelisted_caller();
+		let claimer = caller.clone();
+		let universal_location: UniversalLocationOf<T> = vec![1u8; u as usize].try_into().unwrap();
+		let token_uri: TokenUriOf<T> = vec![1u8; 100usize].try_into().unwrap();
+
+		{
```
Comment (asiniscalchi) [inline pallets/asset-metadata-extender/src/benchmarking.rs:285]: why this block ?

File: pallets/asset-metadata-extender/src/benchmarking.rs [hunk 2]
```diff
@@ -34,6 +124,203 @@ mod benchmarks {
...
```
Comment (asiniscalchi) [inline pallets/asset-metadata-extender/src/benchmarking.rs:223]: ??

File: pallets/asset-metadata-extender/src/benchmarking.rs [hunk 3]
```diff
@@ -34,6 +124,203 @@ mod benchmarks {
...
+		u: Linear<0, { <T as Config>::MaxUniversalLocationLength::get() }>,
+	) {
+		let mut handle = MockHandle::new();
+
+		let caller: T::AccountId = whitelisted_caller();
+		let claimer = caller.clone();
+		let universal_location: UniversalLocationOf<T> = vec![1u8; u as usize].try_into().unwrap();
+		let token_uri: TokenUriOf<T> = vec![1u8; t as usize].try_into().unwrap();
+
+		{
```
Comment (asiniscalchi) [inline pallets/asset-metadata-extender/src/benchmarking.rs:167]: ??

**Review by asiniscalchi — APPROVED** [review]

**Review by asiniscalchi — CHANGES_REQUESTED** [review]
what about the unuseful blocks ?

File: pallets/asset-metadata-extender/src/benchmarking.rs (hunk 1 above)
Comment (magecnion) [reply pallets/asset-metadata-extender/src/benchmarking.rs:285]: if I don't create the extension ahead, `create_token_uri_extension` will fail: https://github.com/freeverseio/laos/blob/a6774a2ca5526c01573864ffd9baf8f5b9a7a617/pallets/asset-metadata-extender/src/precompiles/asset_metadata_extender/mod.rs#L221-L222 and it couldn't be benchmarked

**Review by asiniscalchi — APPROVED** [review]

**Review by asiniscalchi — APPROVED** [review]

**Review by luispdm — APPROVED** [review]

**Pull request merged**